app/views/boards/show.html.erb

```diff
@@ -1,15 +1,23 @@
 <body>
  <div class="container">
  <table class="block-center">
-     <% 4.times do %>
+     <% 4.times do |row| %>
       <tr class="even">
- 	     <% 8.times do %>
- 	      <td></td>	 
+ 	     <% 8.times do |even_cell| %>
+       <% if Piece.where(file: even_cell + 1, rank: row * 2 + 1)[0]%>
```

Comment:
There are two problems with this approach:

- It's looking at all of the pieces in the DB, rather than the pieces associated with a specific game
- In the worst case scenario (a full board), you'll wind up running 64 `where` queries to check each cell and 32 more to get the type associated with each piece, which is very inefficient

The solution is as follows:

- Get rid of the temporary controller/route that you created earlier and make this the view for `GamesController#show`, which will give you access to a game variable (`@game`)
- In the controller, get all of the pieces associated with the game and assign them to an instance variable (e.g. `@game_pieces`). Then, in the view, use your instance variable together with `select` (http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-select) to check whether a piece is present in a specific place and retreive that piece's type. This way, there will be one initial DB query and a bunch of in-memory operations, making this process much more efficient.